docs/getting_started.md

```diff
@@ -216,6 +219,92 @@ Here are some resources that can be used to get you started working with `AnnDat
 - [Converting directly from R to Python](https://theislab.github.io/scanpy-in-R/#converting-from-r-to-python)
 - [Homepage for scanpy tutorials](https://scanpy.readthedocs.io/en/latest/tutorials.html)
 
+
+## Special considerations for CITE-seq experiments
+
+
+If you have ADT data from a CITE-seq experiment, you can use the following commands to access raw and normalized ADT expression matrices in the `processed.rds` file object:
+
+```r
+# the raw ADT counts matrix
+raw_adt_counts <- counts(altExp(processed_sce))
+
+# the log normalized ADT counts matrix
+# if this value is NULL, it means normalization failed
+normalized_adt_counts <- logcounts(altExp(processed_sce))
+```
+
+It is important to be aware that the `SingleCellExperiment` object in the `processed.rds` file was not filtered based on ADT counts, but information that can be used for filtering is  available in the `adt_scpca_filter` column.
```

Comment:
```suggestion
It is important to be aware that the `SingleCellExperiment` object in the `processed.rds` file was not filtered based on ADT counts.
The `adt_scpca_filter` column indicates which cells should be removed before proceeding with downstream analyses of the ADT data.
```
We might also want to say something about what qualifies cells to be in the `adt_scpca_filter` column as remove? I know you link to the processing information, but it might be helpful to have a one line quick summary here and it can't hurt to repeat yourself in docs.